README.md

```diff
@@ -41,6 +45,18 @@ When you run `docker compose up`, Docker Compose automatically combines `docker
 
 In production, the frontend is hosted on Cloudflare Pages, not locally.
 
+### Setup MinIO
+
+You can skip this if you will not develop features related to Problems and Submissions.
+
+Refer to the `docker-compose.override.yml` file for the following configurations:
+
+1. Open the MinIO console at http://localhost:9001 and log in using the username (`MINIO_ROOT_USER`) and password (`MINIO_ROOT_PASSWORD`) specified in the yml file.
+
```

Comment:
I think we should remove extra empty lines between the list items so that it can be rendered as single <ol>.

Reply:
sure, fixed